requirements.txt

```diff
@@ -0,0 +1,6 @@
+elasticsearch==2.1.0
+lxml
+nose
+money
+sqlalchemy>=1.0.6
+psycopg2
```

Comment:
These last few aren't necessary for this function, but getting rid of them would require untangling a lot of imports in core.